src/writer/cluster.h

```diff
@@ -44,10 +44,11 @@ class Cluster {
 
 
   public:
-    Cluster(Compression compression);
+    Cluster(Compression compression, int compression_level);
     virtual ~Cluster();
 
     void setCompression(Compression c) { compression = c; }
+    void setCompressionLevel(int cl) { compressionLevel = cl; }
```

Comment:
We never use `setCompression`.  I'm in favor of removing it instead of adding another `setCompressionLevel`